opensearch/Rakefile

```diff
@@ -55,6 +55,22 @@ namespace :test do
   end
 end
 
+namespace :test_security do
+  desc 'Wait for OpenSearch to be in a green state'
+  task :wait_for_green do
+    sh '../scripts/wait-cluster.sh'
```

Comment:
Is it possible to assume the cluster is already up before executing the test?

Reply:
Without this check, there might be intermittent failures.

Reply:
IMO, it is not responsibility of test runner to wait till cluster is up. It is valid assumption to have cluster up and running before any one runs the test.

Reply:
Ok. Removing the wait time for time being. Will revisit if we observe intermittent failures.